frontend/style.md

```diff
@@ -157,6 +157,8 @@ export const IMPORT_DATASET = gql`
 
 We have two styles when naming functions: `onXXXX` and `handleXXXX`. Note both are camelCase. We use `onXXXX` for functional props, and we use `handleXXXX` when we are locally declaring the function.
 
+Also we have abbreviation like CSV, JSON and HTML. The abbreviation should be integrated into lowercase or uppercase like `xxxCSV` or `csvXXX`.
```

Comment:
First, instead of doing a copy/paste in to function and variable sections, at this point it seems to make more sense to combine the function and variable sections in to a "Naming Conventions" section, since currently both of those sections just talk about how to name.

Then, just a revision:

```suggestion
Abbreviations, like CSV, JSON or HTML, should be EITHER lowercase or uppercase. like `xxxCSV` or `csvXXX`.

<table>
<thead><tr><th>Bad</th><th>Good</th></tr></thead>
<tbody>
<tr><td>
```TypeScript
Html
getCsv
JsonWrapper
```
</td><td>
```TypeScript
html
readJSON
onCSVImport
```
```